utils/currency.ts

```diff
@@ -3,7 +3,8 @@ export type Currency = {
   name: string
   symbol: string
   decimals: number
-  logo: string
+  logo?: string
```

Comment:
Currency should be as vanilla as possible. Let's remove `logo` and `substitute`.